### ISSUE TITLE
Catch 302 redirect on onNavigationStateChange method on Android

### ISSUE THREAD

**Issue by vUdav** [react-native-community/react-native-webview#939]
**Bug description:**
On Android I'm not catch url with 302 status on `onNavigationStateChange` method. On iOS it's work is good.

**To Reproduce:**
[Expo Snack example](https://snack.expo.io/@vudav/webview-302-redirect)

**Expected behavior:**
Get `https://httpstat.us/302` on logs then I use Android.

**Screenshots/Videos:**
![2019-10-08 17 22 11](https://user-images.githubusercontent.com/7684712/66404864-e59a5e00-e9f1-11e9-8fca-d07414e07170.jpg)

**Environment:**
 - OS: Android
 - OS version: 9
 - react-native version: 0.59.9
 - react-native-webview version: 5.12.1

**Comment by awatt-cox** [comment]
I'm having this same issue. on iOS, I see the 302 redirect happening in onNavigationStateChange(), so my custom code can catch the original url (before the redirect) and respond. in Android, I never see the 302:

iOS onNavigationStateChange:
- url 1 is seen by Webview
- 302 redirect happens
- url 2 is seen by Webview

Android onNavigationStateChange:
- url 1 is NOT seen by Webview, so I can't perform any custom actions
- 302 redirect happens
- url 2 IS Seen by Webview

**Comment by MrAlekhin** [comment]
The same issue React Native version = 0.61.2

**Comment by yawnr** [comment]
This really shouldn't be closed -- Android is not receiving the same events that iOS is.

**Comment by fe-tom** [comment]
We've faced exactly the same issue:
iOS was working fine. On Android nothing happens/no errors.

Calling the redirect target url in webview directly was working but the the redirect itself was not executed in webview.

We've fixed this by adding the certificate from our server to android security settings: https://developer.android.com/training/articles/security-config

Then page was showing up. Hope this helps!
